tests/decorators/decorators.go

```diff
@@ -72,6 +72,8 @@ var (
 	RequiresRWXBlock = Label("RequiresRWXBlock")
 	// Requires a storage class with Block storage support
 	RequiresBlockStorage = Label("RequiresBlockStorage")
+	// Requires the containerized-data-importer installed in the cluster
+	RequiresCDI = Label("RequiresCDI")
```

Comment:
so this is basically needed in all the tests that has dvs in them.. which seems like there should be much more then those in this commit..
I get that those are the only tests which bothered to make sure cdi exists but its quite a weird label.. also isnt CDI installed by default with kubevirt? do we have lanes which doesnt install CDI?

Reply:
> do we have lanes which doesnt install CDI?

we don't, should I just drop it? seemed more aggressive

> CDI installed by default with kubevirt?

no, only if it's deployed via uber operators like HCO or others
they're very tightly coupled but nothing stops you from installing the kubevirt operator/cr and stopping there

Reply:
Yeah I meant more like that all our lanes install it by default, so can't see a real reason to actually check it in our tests.. it should just fail if CDI doesnt exist.

Reply:
done